src/static/javascripts/components/dashboard.jsx

```diff
@@ -121,7 +121,7 @@ var MainDiv = React.createClass({
 		if (!loggedIn()) {
 		   return (<Welcome />);
 		}
-        var itemNodes = testQuestions.map(function (item) {
+        var itemNodes = data.map(function (item) {
```

Comment:
That's done/waiting for testing with the back end, I just needed to ask a clarifying question there because normally the line that Carlos changed here is the one that's also distributing out live data cards because that's where the ajax GET directs to. So we're going to need an additional handler here or something if we're having base case (as shown here) + live updates afterwards.